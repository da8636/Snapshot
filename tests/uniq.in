SET a 1 2 2 4 4 4 4 3 3
GET a
UNIQ a
GET a
BYE
