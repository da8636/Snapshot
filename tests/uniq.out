ok

[1 2 2 4 4 4 4 3 3]

ok

[1 2 4 3]

bye
